test-clients/src/main/java/com/hedera/services/bdd/suites/perf/file/MixedFileOpsLoadTest.java

```diff
@@ -73,20 +76,23 @@ protected HapiApiSpec runMixedFileOps() {
 		String targetFile = "targetFile";
 
 		Supplier<HapiSpecOperation[]> mixedFileOpsBurst = () -> new HapiSpecOperation[] {
-				fileCreate(targetFile + submittedSoFar.getAndIncrement()).contents(initialContent),
+				fileCreate(targetFile + submittedSoFar.getAndIncrement())
+						.contents(initialContent)
+						.hasKnownStatusFrom(SUCCESS, UNKNOWN),
 				fileUpdate(targetFile)
 						.fee(ONE_HUNDRED_HBARS)
 						.contents(TxnUtils.randomUtf8Bytes(TxnUtils.BYTES_4K))
 						.noLogging()
 						.payingWith(GENESIS)
 						.hasAnyPrecheck()
+						.hasKnownStatusFrom(SUCCESS, UNKNOWN)
```

Comment:
Better to use a universal list like `defaultAcceptableKnownStatus` with known status that can be used for all tests.

Reply:
That's a good idea. But I am not quite sure of the current `fix`, and hence not feel like to add a new constant list for this one.